PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.webapp/src/com/microsoft/azuretools/webapp/ui/WebAppDeployDialog.java

```diff
@@ -177,6 +179,7 @@ protected Control createDialogArea(Composite parent) {
             @Override
             public void widgetSelected(SelectionEvent e) {
                 sendTelemetry("CREATE");
+                TelemetryUtil.sendTelemetryOpEnd("open create webapp diag", buildProperties());
```

Comment:
why two lines of temeletry?

Reply:
One is the new telemetry schema, another is the old one. Should keep the old one for one or two release version